### ISSUE TITLE
Labels piechart overlapping

### ISSUE THREAD

**Issue by christian-s1** [masayuki0812/c3#1668]
Hi,

I'm using c3.js for a year now and I think it works great, except the following issue:
![image](https://cloud.githubusercontent.com/assets/8776542/14741999/35078054-089a-11e6-8f8d-4ceac5d48b11.png)
As you can see the slices overlap the labels. I know that c3 and d3 work with svg and that svg looks at the order of drawing which element comes in front, but is there a way to get the labels on top? Or a way to move the labels like this for example: 
![image](https://cloud.githubusercontent.com/assets/8776542/14742058/8d78aace-089a-11e6-91f1-546a9edb982f.png)

Greetings,
Christian

**Comment by aendrew** [comment]
Some more control over pie labels would be a good idea. I think in the example you give, part is you have really long data names (your legend takes up a third of the image!).

**Comment by christian-s1** [comment]
Yes, I agree that the labels are to long (the label says that it is way to long in Dutch) But for example the blue one is not that long and it should not be a problem when the labels are drawn over the slices instead of what it is doing now. But you don't know a solution for this problem other than just don't make the labels that long?

**Comment by vitorbaptista** [comment]
@christian-s1 Did you solve the issue with labels being drawn below the pie's slices? I'm having the same problem.

**Comment by tjohannto** [comment]
Would a fix/enhancement like it has been done for flot also possible for C3?

see: 
https://github.com/flot/flot/issues/609
and 
https://github.com/flot/flot/issues/1257



unfortunately I am not familiar enough with C3 to merge this on my own into C3.

**Comment by tjohannto** [comment]
I hope it is OK to add another comment here.
I have tried to apply the fix from "flot" to manually move the pie labels:

http://jsfiddle.net/gothmogg/fo5xe1nt/

You can of course use this but I would also love to get any feedback why this does not work as expected. Is the middle of the pie not [0,0]?
